engine/src/main/java/org/terasology/rendering/dag/nodes/SimpleBlendMaterialsNode.java

```diff
@@ -74,23 +76,11 @@ public void process() {
      */
     private void preRenderSetupSimpleBlendMaterials() {
         READ_ONLY_GBUFFER.setRenderBufferMask(true, true, true);
-        GL11.glEnable(GL_BLEND);
         GL11.glBlendFunc(GL_SRC_ALPHA, GL_ONE_MINUS_SRC_ALPHA); // (*)
```

Comment:
Finally, make sure to move the javadoc for this method somewhere else rather than delete it, as it has useful information.